addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -11,6 +11,20 @@
 from odoo.addons.phone_validation.lib import phonenumbers_patch
 
 class TestPhonenumbersPatch(BaseCase):
+
+    def _assert_region_numbers_validity(self, region_code:str, numbers:list):
+        """Helper function checking validity of all given phone numbers for a given region.
+        Args:
+            region_code (str): ex. 'MA' for Morroco
+            numbers (list): list of strings representing phonenumbers that should be valid
+        """
```

Comment:
Love to see these.

Linter doesn't seem to care but so far all type hints have a space before the type so let's keep it this way from the get-go.

Also we generally use `:param type name: description`
I don't think we generate any documentation from it but just in case let's stick with common syntax.

Reply:
Thanks for the hint, included